public/app/features/profile/ChangePasswordForm.tsx

```diff
@@ -53,7 +52,8 @@ export const ChangePasswordForm = ({ user, onChangePassword, isSaving }: Props)
       `}
     >
       <Form onSubmit={onChangePassword}>
```

Comment:
```suggestion
      <Form onSubmit={onChangePassword} maxWidth={400}>
```
Not related to your PR, but could you remove the wrapper div and put the max width on the Form itself. Also could you update the Form import to be `import { Form } from 'app/core/components/Form/Form';` as importing it from grafana/ui is deprecated?

Reply:
Changes as requested